src/phoenix.js

```diff
@@ -113,16 +112,6 @@ function loadApps () {
     router,
     render: h => h(Phoenix)
   })
-
-
-  // inject custom theme config into vuex
-  fetch(`themes/${config.theme}.json`)
```

Comment:
When we get rid of this, we are also losing non-style related stuff - name that's displayed on login and next to the exit in menu, slogan ... Maybe just remove the style mentions instead of the whole? Or do you plan to include these strings in different way?

Reply:
Good intervention. Having strings to display in a name-of-theme.json is important. I'll update my approach.